opentracing_instrumentation/client_hooks/requests.py

```diff
@@ -25,44 +25,55 @@
 import logging
 import urllib.parse
 
+from opentracing.ext import tags
 from ..request_context import get_current_span
 from ..http_client import AbstractRequestWrapper
 from ..http_client import before_http_request
 from ..http_client import split_host_and_port
-from ._singleton import singleton
+from .base import Patcher
 
 log = logging.getLogger(__name__)
 
 # Try to save the original entry points
 try:
-    import requests.sessions
     import requests.adapters
 except ImportError:  # pragma: no cover
     pass
 else:
     _HTTPAdapter_send = requests.adapters.HTTPAdapter.send
 
 
-@singleton
-def install_patches():
```

Comment:
can we keep this as deprecated for backwards compatibility? There was no requirement previously that everything must go through install_all_patches.

Reply:
Sure. Should I add some warning or just a comment is enough?

Reply:
just a comment, it's unusual for people to call this anyway

Reply:
It's done.